step.yml

```diff
@@ -1,14 +1,35 @@
 title: Run CocoaPods install
 summary: |-
-  You don't need to keep the installed pods in your repo, simply add this step
+  You don't need to keep the installed pods in your repo. Simply add this Step
   and Bitrise will do the pod install for you on the VM!
 description: |-
-  Run CocoaPods install (`$ pod install`) in your App's directory.
 
-  CocoaPods version is determined based on the Podfile.lock or
-  on the Gemfile.lock (if Podfile.lock does not exist in the Podfile's
-  directory and Gemfile.lock contains cocoapods gem)
-  in the Podfile's directory.
+  CocoaPods is a dependency manager for Swift and Objective-C Cocoa projects. This Step uses CocoaPods' `pod install` command to install your dependencies on the virtual machine where your Bitrise build runs. CocoaPods version is determined based on the Podfile.lock file or on the Gemfile.lock file if Podfile.lock does not exist in the Podfile's
```

Comment:
"Swift and Objective-C Xcode projects" makes more sense as Cocoa is a macOS UI framework only.

Reply:
That was supposed to be simply "Swift and Objective-C projects", no idea what Cocoa is doing there or how it got there :)